### ISSUE TITLE
Using FAS for fedorapeople in create-review

### ISSUE THREAD

**Issue by LyesSaadi** [juhp/fbrnch#16]
Hello!

First of all, great app! I've been using it for some time now, and it really helps!

Anyway, when trying to create a review, I get this message:
```
lyes@fedorapeople.org: Permission denied (publickey).
fbrnch: ssh fedorapeople.org mkdir -p public_html/reviews/cLaTeXMath failed with exit code 255
```

The problem is in `lyes@fedorapeople.org`, which is the wrong FAS... Indeed, my FAS is different from my username... When attempting to connect via ssh, it uses the username by default, while it should use my FAS.

Looking at the code, I'm not a Haskell developer, but I can see that it already uses my FAS for checking that the files were created properly !

**Comment by juhp** [comment]
Thank you for the report, Lyes

> First of all, great app! I've been using it for some time now, and it really helps!

Thanks, that's good to hear :-)

> Anyway, when trying to create a review, I get this message:
> ```
> lyes@fedorapeople.org: Permission denied (publickey).
> fbrnch: ssh fedorapeople.org mkdir -p public_html/reviews/cLaTeXMath failed with exit code 255
> ```
> The problem is in `lyes@fedorapeople.org`, which is the wrong FAS... Indeed, my FAS is different from my username... When attempting to connect via ssh, it uses the username by default, while it should use my FAS.
> 
> Looking at the code, I'm not a Haskell developer, but I can see that it already uses my FAS for checking that the files were created properly !

Yes you are right - seems I was lazy when I wrote this code originally...

Thanks so much for reporting this.

Should get fixed in the next release.

**Comment by juhp** [comment]
I released and built 0.6.9
